terraform/terraform.tfstate

```diff
@@ -0,0 +1,279 @@
+{
+    "version": 3,
+    "terraform_version": "0.11.4",
+    "serial": 12,
+    "lineage": "5b5822db-1d83-13de-76ea-0dd99b442767",
```

Comment:
⚠️ This is bad - we need to ignore terraform state and may need to change our account to protect what's in this one.

Reply:
I guess it's moderately safe because we don't use any database passwords or s3 bucket keys.

Reply:
I double checked for any secrets before committing - seems like it's actually a convenient way to share local state?